spec/spec_helper.rb

```diff
@@ -32,6 +32,10 @@ def get_plugin(plugin, ohai = Ohai::System.new, path = PLUGIN_PATH)
   loader.load_plugin(File.join(path, "#{plugin}.rb"))
 end
 
+def convert_window_output(stdout)
```

Comment:
This wasn't what I was supposed to do?
Le 24 mars 2015 15:24, "Bryan McLellan" <notifications@github.com> a écrit :

> In spec/spec_helper.rb
> <https://github.com/chef/ohai/pull/474#discussion_r27031718>:
>
> > @@ -32,6 +32,10 @@ def get_plugin(plugin, ohai = Ohai::System.new, path = PLUGIN_PATH)
> >    loader.load_plugin(File.join(path, "#{plugin}.rb"))
> >  end
> >
> > +def convert_window_output(stdout)
>
> convert_windows_output?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/ohai/pull/474/files#r27031718>.
>